src/Client/Web/Playground/Bit.Client.Web.BlazorUI.Playground/Web/Pages/Components/ChoiceGroup/BitChoiceGroupDemo.razor.cs

```diff
@@ -0,0 +1,485 @@
+                    SelectedImageSrc = "https://static2.sharepointonline.com/files/fabric/office-ui-fabric-react-assets/choicegroup-pie-selected.png",
+                    ImageAlt = "alt for Pie image",
+                    ImageSize = new System.Drawing.Size(32, 32)
+                }
+            };
+
+            Example4Options = new()
+            {
+                new BitChoiceGroupOption()
+                {
+                   Text = "Day",
+                   Value = "Day",
+                   IconName = BitIconName.CalendarDay
+                },
+                new BitChoiceGroupOption()
+                {
+                    Text = "Week",
+                    Value = "Week",
+                    IconName = BitIconName.CalendarWeek
+                },
+                new BitChoiceGroupOption()
+                {
+                    Text = "Month",
+                    Value = "Month",
+                    IconName = BitIconName.Calendar,
+                    IsEnabled = false
+                }
+            };
+
+            Example7Options = new()
+            {
+                new BitChoiceGroupOption()
+                {
+                   Text = "Option A",
+                   Value = "A",
+                   OnChange = () => {OptionOnChange("A"); }
+                },
+                new BitChoiceGroupOption()
+                {
+                    Text = "Option B",
+                    Value = "B",
+                    OnChange = () => {OptionOnChange("B"); }
+                },
+                new BitChoiceGroupOption()
+                {
+                   Text = "Option C",
+                   Value = "C",
+                   OnChange = () => {OptionOnChange("C"); }
+                },
+                new BitChoiceGroupOption()
+                {
+                   Text = "Option D",
+                   Value = "D",
+                   OnChange = () => {OptionOnChange("D"); }
+                }
+            };
+        }
+
+        public class ChoiceGroupValidationModel
+        {
+            [Required(ErrorMessage = "Pick one")]
+            public string Value { get; set; }
+        }
+        public ChoiceGroupValidationModel ValidationModel = new();
+        public string SuccessMessage { get; set; } = string.Empty;
+
+        public string OnChangeValue { get; set; } = string.Empty;
+
+        private readonly List<ComponentParameter> componentParameters = new()
+        {
+            new ComponentParameter
+            {
+                Name = "Options",
+                Type = "List<BitChoiceGroupOption>",
+                DefaultValue = "new List<BitChoiceGroupOption>()",
+                Description = "List of options, each of which is a selection in the ChoiceGroup."
+            },
+            new ComponentParameter
+            {
+                Name = "Name",
+                Type = "string",
+                DefaultValue = "Guid.NewGuid().ToString()",
```

Comment:
I think this style of introducing the default value is not correct. check it by @msynk

Reply:
I don't understand what is the problem here.

Reply:
i think its better to use a phrase like this: `a Guid`